genanki/note.py

```diff
@@ -7,22 +7,17 @@ class Note:
   def __init__(self, model=None, fields=None, sort_field=None, tags=None, guid=None):
     self.model = model
     self.fields = fields
-    self.sort_field = sort_field
+    if sort_field == None and model != None:
+        self.sort_field_value = fields[model.sort_field_id]
+    else:
+        self.sort_field_value = sort_field  # overwrite model defined value if sort_field is passed
     self.tags = tags or []
     try:
       self.guid = guid
     except AttributeError:
       # guid was defined as a property
       pass
 
-  @property
```

Comment:
Removing the `sort_field` property is a backwards-incompatible change. Could you rework the PR to preserve the `.sort_field` property?

Reply:
Sure yes! I did not realise this was breaking backward compatibility sorry. I never used python `@property` before. I'll force-push a commit, rebasing on top of your `master` branch and squashing the fix.